### PR TITLE
Simplify web app Dockerfile

### PR DESCRIPTION
**Pull request by LinusU**
1) Remove redundant call to `mkdir -p` since the `WORKDIR` command creates the directory.
2) Uses relative targets in the `COPY` command to leverage the previous call to `WORKDIR`

### CODE REVIEW

File: locale/en/docs/guides/nodejs-docker-webapp.md [hunk 1]
```diff
@@ -99,7 +98,7 @@ need to do is to install your app dependencies using the `npm` binary:
 
 ```docker
 # Install app dependencies
-COPY package.json /usr/src/app/
+COPY package.json ./
```
Comment (fhemberger) [inline locale/en/docs/guides/nodejs-docker-webapp.md]: `./` not required `.` should be enough

File: locale/en/docs/guides/nodejs-docker-webapp.md [hunk 2]
```diff
@@ -132,15 +131,14 @@ Your `Dockerfile` should now look like this:
 FROM node:boron
 
 # Create app directory
-RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
 # Install app dependencies
-COPY package.json /usr/src/app/
+COPY package.json ./
```
Comment (fhemberger) [inline locale/en/docs/guides/nodejs-docker-webapp.md]: same here

**Review by fhemberger — CHANGES_REQUESTED** [review]
A little nitpicking, otherwise LGTM.

**Review by fhemberger — APPROVED** [review]

**Comment by fhemberger** [comment]
Thank you!

**Comment by LinusU** [comment]
Thanks for fixing the nitpicks! been busy packing my apartment into boxes these last few days 🙈

**Pull request merged**